lab-davide/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-davide",
+  "version": "1.0.0",
+  "description": "",
+  "main": "hello.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FazioD/lab-02-build-tools.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FazioD/lab-02-build-tools/issues"
+  },
+  "homepage": "https://github.com/FazioD/lab-02-build-tools#readme",
+  "devDependencies": {
+    "eslint-watch": "^2.1.10",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^2.0.0",
+    "gulp-mocha": "^2.2.0"
```

Comment:
add "mocha" to your devDependencies and remove "eslint-watch" you should be using the built in gulp watch method :)